lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -560,6 +560,27 @@ export class UserAgentApplication {
         });
     }
 
+    /**
+     * API interfacing idToken request when applications already have a session/hint acquired by authorization client applications
+     * @param request
+     */
+    ssoSilent(request: AuthenticationParameters): Promise<AuthResponse> {
```

Comment:
probably want to look at this holistically in the lib moving forward, but have you considered logging here? 

Are there things we can add at a basic level that will help us debug if there are any issues with this moving forward?